.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
-.github/CODEOWNERS @sonarsource/devex-devandteamworkflow-squad
+.github/CODEOWNERS @sonarsource/orchestration-workflow-squad
```

Comment:
there is a problem here, we may need Dev infra to add the new squad (unless there is a typo)